util/init_context.go

```diff
@@ -53,7 +54,18 @@ func InitUDMContext(udmContext *context.UDMContext) {
 	udmContext.NrfUri = configuration.NrfUri
 	servingNameList := configuration.ServiceNameList
 
-	udmContext.Keys = configuration.Keys
+	udmContext.SuciProfiles = []suci.SuciProfile{
+		{
+			ProtectionScheme: "1",
+			PrivateKey:       configuration.Keys.UdmProfileAHNPrivateKey,
+			PublicKey:        configuration.Keys.UdmProfileAHNPublicKey,
+		},
+		{
+			ProtectionScheme: "2",
```

Comment:
Is it worth mentioning why `ProtectionScheme: "2"` (standard defined value for `Profile <B>`)? Or add a reference to TS 33.501 Annex C.1?